app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -128,17 +154,27 @@ protected Void doInBackground(Mood ... moods ) {
                                 }
                                 break;
                             case 5:
-                                query += ("\"image\" : \"" + mood.getImage() + "\"");
+                                if(mood.getImage() == null) {
+                                    query += ("\"image\" : " + mood.getImage());
+                                }
+                                else {
+                                    query += ("\"image\" : \"" + mood.getImage() + "\"");
+                                }
```

Comment:
We store a null field in elasticsearch if the image is null now. A null field, not a String that says "null".